src/components/EstimatedLifecycleCost/Review/index.test.tsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+import { defaultEstimatedLifecycle } from 'data/businessCase';
+import EstimatedLifecycleCostReview from './index';
+
+// Get past TS errors
```

Comment:
What TS errors are you seeing? I'd love to figure out how to remove this unnecessary code

Reply:
`Property 'matchMedia' does not exist on type 'Global'.`

The test still runs and passes without declaring `global`. I guess it's more of a mechanism to get rid of the rid squiggly lines.